.gitignore

```diff
@@ -1,5 +1,6 @@
 # Environment variables
-.env
+# why .env was blacklisted? it is useful for the docker-compose.yaml testing file.
+# .env
```

Comment:
.env should be blacklisted and the .env above should be called env.example or something similar.

Reply:
The purpose of docker-compose.yaml is testing, so it can contain all testing env vars directly